### PR TITLE
Hl-827, HL-816, HL-824, HL-793: New applicant login page and Accessibility statement

### PR DESCRIPTION
**Pull request by mjturt**
## Description :sparkles:

- HL-827: New applicant login page style
- HL-816: Accessibility Statement page
- HL-824: Make PDF links clickable
- HL-793: Fix accessibility statement link

The accessibility statement is only in Finnish. Login page texts should be translated by a professional later.

### CODE REVIEW

File: frontend/benefit/applicant/src/components/pages/Pages.sc.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export const $Hr = styled.hr`
+  border: none;
+  border-top: 1px solid ${(props) => props.theme.colors.black20};
+  margin-top: ${(props) => props.theme.spacing.l};
+  margin-bottom: ${(props) => props.theme.spacing.l};
+  width: 100%;
+`;
+
+export const $H1 = styled.h1`
```
Comment (sirtawast) [inline frontend/benefit/applicant/src/components/pages/Pages.sc.ts:11]: These are named `Heading` and `Subheading`, makes refactoring easier.

File: frontend/benefit/applicant/src/components/pages/Pages.sc.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  font-size: ${(props) => props.theme.fontSize.heading.l};
+  font-weight: 300;
+`;
+
+export const $P = styled.p`
+  line-height: ${(props) => props.theme.lineHeight.l};
+`;
+
+export const $H2 = styled.h2`
```
Comment (sirtawast) [inline frontend/benefit/applicant/src/components/pages/Pages.sc.ts:20]: These are named `Heading` and `Subheading`, makes refactoring easier.

File: frontend/benefit/applicant/public/locales/fi/common.json [hunk 3]
```diff
@@ -468,14 +468,22 @@
...
     "errorLabel": "Tapahtui tuntematon virhe. Kirjaudu uudelleen sisään",
     "sessionExpiredLabel": "Käyttäjäsessio vanhentui. Kirjaudu uudelleen sisään",
-    "infoLabel": "Palvelun käyttäminen edellyttää vahvaa tunnistautumista",
+    "infoLabel": "Ongelmia kirjautumisessa?",
     "logoutInfoContent": "Sinut on kirjattu ulos. Jos haluat jatkaa asiointia palvelussa, kirjaudu uudelleen sisään klikkaamalla alla olevasta painikkeesta.",
-    "infoContent": "Jotta voit edetä palvelussa, sinun täytyy tunnistautua. Voit valita itse, mitä tunnistustapaa haluat käyttää. Tunnistustietosi kulkevat suojatussa yhteydessä. Klikkaa alla olevasta painikkeesta tunnistautuaksesi.",
-    "termsOfServiceHeader": "Tietoa työnantajan edustajien henkilötietojen käsittelystä"
+    "infoContent": "Lähetä meille sähköposti osoitteeseen helsinkilisa@hel.fi.",
+    "termsOfServiceHeader": "Tietoa työnantajan edustajien henkilötietojen käsittelystä",
+    "header": "Kirjaudu Helsinki-lisän asiointipalveluun",
```
Comment (sirtawast) [inline frontend/benefit/applicant/public/locales/fi/common.json:479]: `header` → `heading` and `smallHeader` → `subHeading` for consistency throughout the apps.

File: frontend/benefit/applicant/public/locales/en/common.json [hunk 4]
```diff
@@ -564,6 +564,98 @@
     "home": "Helsinki benefit - Frontpage",
     "createApplication": "Helsinki benefit - Application - Create",
     "editApplication": "Helsinki benefit - Application - Edit",
-    "viewApplication": "Helsinki benefit - Application - View"
+    "viewApplication": "Helsinki benefit - Application - View",
+    "accessibilityStatement": "Helsinki benefit - Accessibility statement"
+  },
+  "accessibilityStatement": {
```
Comment (sirtawast) [inline frontend/benefit/applicant/public/locales/en/common.json:570]: Seems to be just basic HTML altogether. Could this part be just included from HTML file or just use `dangerouslySetInnerHTML`?

File: frontend/benefit/applicant/src/constants.ts [hunk 5]
```diff
@@ -12,6 +12,7 @@ export enum ROUTES {
   APPLICATION_FORM = '/application',
   LOGIN = '/login',
   TERMS_OF_SERVICE = '/terms-of-service',
+  ACCESSIBILITY_STATEMENT = '/accessibility-statement',
```
Comment (sirtawast) [inline frontend/benefit/applicant/src/constants.ts:15]: On a side note, should these be Finnish by default? Then maybe translate them later?

File: frontend/benefit/applicant/src/hooks/useUserQuery.ts [hunk 6]
```diff
@@ -41,7 +51,7 @@ const useUserQuery = (
     () => handleResponse<UserData>(axios.get(BackendEndpoint.USER_ME)),
     {
       refetchInterval: FIVE_MINUTES,
-      enabled: !logout,
+      enabled: isEnabled(),
```
Comment (sirtawast) [inline frontend/benefit/applicant/src/hooks/useUserQuery.ts:54]: Not sure about the side-effects of this, namely the calling for `/me` throughout the apps. Should be fine but let's keep this in mind if app halts to an error fiesta.

**Review by sirtawast — CHANGES_REQUESTED** [review]
* Requesting some changes mainly related to naming conventions / inconsistancies and practices. 
* Inserting the accessibility statement in plain HTML could be beneficial for future edits (no react skills required, faster edits ...)
* Some notes that might be worthy if troubles arise (looking at userQuery hook). 
 
Otherwise, looks good to me.

File: frontend/benefit/applicant/public/locales/en/common.json (hunk 4 above)
Comment (mjturt) [reply frontend/benefit/applicant/public/locales/en/common.json:570]: Sure, it's just basic HTML. How about the translations then?

File: frontend/benefit/applicant/src/constants.ts (hunk 5 above)
Comment (mjturt) [reply frontend/benefit/applicant/src/constants.ts:15]: We have a URL schema which includes the language part, e.g. `https://helsinkilisa.hel.fi/fi/login` and `https://helsinkilisa.hel.fi/en/login`, which seems to be common practice in other Helsinki sites. Another common pattern is to translate all routes, which, I think, is a bit better, but we lack support for that currently.

File: frontend/benefit/applicant/src/hooks/useUserQuery.ts (hunk 6 above)
Comment (mjturt) [reply frontend/benefit/applicant/src/hooks/useUserQuery.ts:54]: If the `/me` call fails, the app redirects to the login page. Accessibility statement needs to be seen by unauthenticated users too. This logic could be transferred to the `handleError` function, though.

**Review by rikuke — APPROVED** [review]
Looks good to me!

**Review by sirtawast — APPROVED** [review]

**Pull request merged**